config/common/server.yaml

```diff
@@ -126,8 +166,9 @@ deploymentConfigs:
   - metadata:
       name: "{{.ApplicationName}}-postgresql-{{$index}}"
       labels:
-        app: "{{.ApplicationName}}"
-        service: "{{.ApplicationName}}-postgresql-{{$index}}"
+        app: "{{.ApplicationName}}-postgresql-{{$index}}"
+        application: "{{.ApplicationName}}"
+        service: "{{.ApplicationName}}-postgresql"
```

Comment:
not all kieservers will use postgres. i actually think only prod env uses it.

Reply:
The authoring.yml file was deploying postgres, that’s why I made the kieserver use it

Reply:
OK ... @bmozaffa are we ok w/ that? I'm not sure authoring-ha uses it. Does prod-immutable?

Reply:
Ok, so we have 5 templates... if 3 or 4 of them use something, we should make it common and delete it in the others. Otherwise, not make it common.

@ruromero I don't see authoring use Postgres in the latest template (under CE repo). It uses H2 there, and it's only prod/immutable, so I'd say to skip.

Reply:
@bmozaffa you're right. I don't know where I took the postgres deployment from. My bad. I'll go back to h2 or maybe make a copy for the moment just to not lose the postgres config (it took me a while)